kernels/datascience-notebook/Dockerfile

```diff
@@ -0,0 +1,116 @@
+# syntax = docker/dockerfile:1.4.1
+ARG BASE_IMAGE=jupyter/datascience-notebook
+ARG PYTHON_VERSION=3.9.6
```

Comment:
```suggestion
ARG PYTHON_VERSION=3.9.14
```

Reply:
`3.9.13` is the latest jupyter/datascience-notebook